lib/colors.ts

```diff
@@ -153,6 +154,27 @@ export function toPercent(value: number): string {
   return `${clean(100 * value)}%`
 }
 
+function isHexNotation(value: string): boolean {
+  return /^#?([\da-f]{3}|[\da-f]{4}|[\da-f]{6}|[\da-f]{8})$/i.test(value)
+}
+
+const OUTPUT_FORMAT_MAP: Partial<Record<Color['mode'], OutputFormats>> = {
+  hsl: 'hsl',
+  lab: 'lab',
+  lch: 'lch',
+  oklab: 'oklab',
+  oklch: 'numbers',
+  p3: 'p3',
+  rgb: 'rgb'
+}
+
+export function toOutputFormat(value: string): OutputFormats | undefined {
+  let parsed = parseAnything(value)
+  if (!parsed) return undefined
+  if (isHexNotation(value)) return 'hex'
+  return OUTPUT_FORMAT_MAP[parsed.mode] ?? 'hex/rgba'
```

Comment:
```suggestion
  return OUTPUT_FORMAT_MAP[parsed.mode] ?? undefined
```

The default behavior should be not changing format

Reply:
Fixed